skimage/feature/_kb.py

```diff
@@ -0,0 +1,264 @@
+from __future__ import print_function, division
+import numpy as np
+from scipy import ndimage as ndi
+from ..util import img_as_ubyte
+from .._shared.utils import assert_nD
+
+
+def kb(image, min_scale=10, max_scale=25, saliency_threshold=0.6, clustering_threshold=7):
+    """Finds salient regions in the given grayscale image.For each point x
```

Comment:
This should be a single sentence followed by a blank line, and should be in present imperative ("Finds salient regions" -> "Find salient regions"). Please review https://www.python.org/dev/peps/pep-0257/  and apply here.